drivers/frequency/ad9517/ad9517.c

```diff
@@ -153,11 +153,26 @@ int32_t ad9517_setup(struct ad9517_dev **device,
 			return ret;
 	}
 	/* Check if VCO is selected as input. */
-	if(dev->ad9517_st.pdata->vco_clk_sel)
+	if(dev->ad9517_st.pdata->vco_clk_sel) {
 		/* Sets the VCO frequency. */
 		ad9517_vco_frequency(dev,
 				     dev->ad9517_st.pdata->int_vco_freq);
 
+		/* Activate PLL */
+		reg_value = AD9517_PLL_POWER_DOWN(0) |
+			    AD9517_CP_MODE (3) |
+			    AD9517_CP_CURRENT (5) |
```

Comment:
I understand that PLL_POWER_DOWN must be zero because you want to enable PLL, that's clear.
But if you compare the CP_MORE to the datasheet, it matches the default value in the register.
Yet the CP_CURRENT is 0x5 and in the datasheet is 0x7 by default.
We should stick to default values unless we add a way to configure this setting or there is a reasonable argument to go with different setting by default.

Reply:
Yes, CP_CURRENT should be 0x7, I misread it from the datasheet.

Reply:
Ok, just change that and you have an approve from me, thanks for contributing !

Reply:
Just make sure the astyle.sh is performed, you probably saw there is an automatic test for that that currently fails.

Reply:
Yes, that's very weird, it succeeded before I executed astyle.sh, now I did it and it fails...

Reply:
I think it might have to do with branch not being in sync with master ? Try rebasing your branch on top of latest analogdevicesinc/master, I think astyle rule will pass after that.